### ISSUE TITLE
Docs Site is Breaking ❌ Important

### ISSUE THREAD

**Issue by Vijayabaskar56** [gluestack/gluestack-ui#2506]
### Description

When I try to Navigate between differnt Pages between docs it was Breaking

### CodeSandbox/Snack link

https://drive.google.com/file/d/1P-Iqnwy_V3DMCc01enD8oiGFNgRjX0Dn/view?usp=sharing

### Steps to reproduce

2. Click on '...' 
3. Scroll down to '...' 

### gluestack-ui Version

v2

### Platform

- [ ] Expo
- [ ] React Native CLI
- [ ] Next
- [X] Web
- [ ] Android
- [ ] iOS

**Comment by Viraj-10** [comment]
Hey @Vijayabaskar56, which browser are you using? Could you please check it in incognito mode?

**Comment by Vijayabaskar56** [comment]
![Screenshot 2024-10-07 181837](https://github.com/user-attachments/assets/d79b636c-553d-4702-8786-e7e1891e84e1)
Here you see in chrome also it was breaking

**Comment by Viraj-10** [comment]
Can you try it in `incognito mode` ? There might be some extension or blocker which might be causing this issue.  
<img width="1435" alt="Screenshot 2024-10-09 at 10 41 20 AM" src="https://github.com/user-attachments/assets/6aee3465-3208-4404-a168-82bf855290ae">

**Comment by Vijayabaskar56** [comment]
Yeah, it is not happening in Incognito Mode, Might be Issue With any of Extension, 
![image](https://github.com/user-attachments/assets/6717aacb-aa1c-438b-b7a0-f88d6414cd22)
anyway here are some log , that might help you

**Comment by Viraj-10** [comment]
Hey @Vijayabaskar56, Can you identify extension which is causing this issue. We are unable to reproduce this issue.

**Comment by Vijayabaskar56** [comment]
These are the common extension between my chrome and arc
IDM Integration Module - 6.42.22
Awesome Screen Recorder & Screenshot - 4.4.22
